shortcircuit-api/src/main/java/com/powsybl/shortcircuit/option/FaultOptions.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.shortcircuit.option;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.powsybl.commons.json.JsonUtil;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class FaultOptions {
```

Comment:
I don't understand this design, FaultOptions could only be linked to one Fault through a FaultContext:
  - FaultContext is a useless object only made of one ID field
  - As it is not embeded in Fault class, we need to resolve link between Fault and FaultOption after using a utility class FaultOptionsIndex which is painful.
 
For me this is a very poor design API that will be painful to use:

Why not just embed all these parameters in the fault?
I would understand to have it separated in case of multiple same options associated to a set of fault, but here this is not the case. We have just a one to one relation between Fault and FaultOptions.